examples/PlugInSamples/NoiseGate/CMakeLists.txt

```diff
@@ -0,0 +1,186 @@
+# Copyright (c) 2016 Matthieu Talbot
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+cmake_minimum_required(VERSION 3.0)
+
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/../../../cmake")
+include(Reprojucer)
+
+
+get_filename_component(JUCE_ROOT "${CMAKE_CURRENT_LIST_DIR}/../../../../JUCE" ABSOLUTE CACHE)
+
+get_filename_component(NoiseGate_jucer_FILE
+  "${JUCE_ROOT}/examples/PlugInSamples/NoiseGate/NoiseGate.jucer" ABSOLUTE CACHE
+)
+
+if(NOT EXISTS "${NoiseGate_jucer_FILE}")
+  message(FATAL_ERROR "No such file: ${NoiseGate_jucer_FILE}")
+endif()
+
+get_filename_component(NoiseGate_jucer_DIR "${NoiseGate_jucer_FILE}" DIRECTORY)
+
+
+
+jucer_project_begin(
+  PROJECT_NAME "NoiseGate"
+  PROJECT_VERSION "1.0.0"
+  COMPANY_NAME "ROLI Ltd."
+  COMPANY_WEBSITE "www.juce.com"
+  COMPANY_EMAIL "info@juce.com"
+  PROJECT_TYPE "Audio Plug-in"
+  BUNDLE_IDENTIFIER "com.ROLI.NoiseGate"
+  BINARYDATACPP_SIZE_LIMIT "Default"
+  # BINARYDATA_NAMESPACE
+  # PREPROCESSOR_DEFINITIONS
+  PROJECT_ID "kmYCMo"
+)
+
+jucer_audio_plugin_settings(
+  BUILD_VST ON
+  BUILD_AUDIOUNIT ON
+  PLUGIN_NAME "NoiseGate"
+  PLUGIN_DESCRIPTION "NoiseGate"
```

Comment:
@MartyLake: in https://github.com/julianstorer/JUCE/blob/master/examples/PlugInSamples/NoiseGate/NoiseGate.jucer#L6, the plugin name and description are `Noise Gate`.